Makefile

```diff
@@ -40,6 +40,7 @@ DOCKER_ENVS := \
 	-e no_proxy
 # note: we _cannot_ add "-e DOCKER_BUILDTAGS" here because even if it's unset in the shell, that would shadow the "ENV DOCKER_BUILDTAGS" set in our Dockerfile, which is very important for our official builds
 
+SEND_CONTEXT := $(if $(BIND_DIR),,$(if $(BINDDIR),,1))
```

Comment:
Let me think; I think the issue was that that the _original_ `BIND_DIR` is overwritten at line 47, and will be either `$BINDDIR` or `"bundles"`. If it's `"bundles"`, the source-code is not bind-mounted. So we need to look at what's provided at the command-line; if `BINDDIR=.` or `BIND_DIR=.` is provided, _then_ the intent was to bind-mount the source files.

I'll check once more